### PR TITLE
feat: Implement LIC8 with tests. Closes #15

### LINKED ISSUE #15: LIC 8
There exists at least one set of three data points separated by exactly A PTS and B PTS
consecutive intervening points, respectively, that cannot be contained within or on a circle of
radius RADIUS1. The condition is not met when NUMPOINTS < 5.
1 ≤ A PTS, 1 ≤ B PTS
A PTS + B PTS ≤ (NUMPOINTS − 3)

### PR DESCRIPTION
**Pull request by coffecup25**

### CODE REVIEW

File: src/main/CMV.java [hunk 1]
```diff
@@ -104,7 +104,33 @@ private Boolean lic7_calculate() {
...
+    private boolean lic8_calculate() {
+        //The condition is not met when NUMPOINTS < 5.
+        if(this.datapoints.length < 5){
+            return false;
+        }
+
+        if (Parameters.A_PTS < 1 || Parameters.B_PTS < 1 ||
+                Parameters.A_PTS+Parameters.B_PTS > (datapoints.length-3)){
+            return false;
+        }
```
Comment (a-kbd) [inline src/main/CMV.java:116]: discussion on discord, also issue #63, you could then safely remove this

**Review by a-kbd — APPROVED** [review]
Looks nice, well done! 

However, see comment and add an issue to change lines affected once issue #63 is resolved

File: src/main/CMV.java (hunk 1 above)
Comment (coffecup25) [reply src/main/CMV.java]: Resolved by commit 8131156

**Comment by coffecup25** [comment]
closes #81 as well

**Pull request merged**